demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -23,15 +23,22 @@
 struct ssl_id {
     SSL *s;
     uint64_t id;
+    int status; /* 0, CLIENTUNIOPEN or CLIENTUNIOPEN|CLIENTCLOSED */
 };
+#define CLIENTUNIOPEN 0x01
+#define CLIENTCLOSED  0x02
 
 #define MAXSSL_IDS 20
 struct h3ssl {
```

Comment:
If you have some cycles, it would be nice to have some comments explaining what the fields of this structure are for, the lifecycle of a client connection and its related "ids", ...

That said, this is about moving the code in the direction of becoming a tutorial, not just an API demo, so ultimately fleshing that out will fall on the core team. Still, I'm hoping you can leave some more "breadcrumbs" for the uninitiated.

Reply:
I have added some comments. I hope that will help.